device/arista/x86_64-arista_7050_qx32/Arista-7050-QX32/buffers.json.j2

```diff
@@ -90,7 +90,7 @@
             "xon":"18432",
             "xoff":"40560",
             "size":"41808",
-            "dynamic_th":"-4",
+            "dynamic_th":"-3",
```

Comment:
there is dynamic_th in pg_profile_lookup.ini, why do we have two places to for alpha?

Reply:
this hwsku and s6000 have not used pg_profile lookup. only Arista-7050-QX-32S is using the lookup method. Will update the former two.